tutorials/assets_pipeline/importing_images.rst

```diff
@@ -40,183 +41,500 @@ Importing textures
+| **Quality loss** | |good| None                   | |regular| Slight     | |bad| Moderate                                       | |good| None            | |bad| Moderate                       |
++------------------+-------------------------------+----------------------+------------------------------------------------------+------------------------+--------------------------------------+
+| **Load time**    | |bad| Slow                    | |bad| Slow           | |good| Fast                                          | |regular| Normal       | |regular| Normal                     |
++------------------+-------------------------------+----------------------+------------------------------------------------------+------------------------+--------------------------------------+
 
 .. |bad| image:: img/bad.png
 
 .. |good| image:: img/good.png
 
 .. |regular| image:: img/regular.png
 
-HDR Mode
-~~~~~~~~
+Estimated memory usage for a single RGBA8 texture with mipmaps enabled:
+
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| Texture size  | Lossless            | Lossy               | VRAM Compressed     | VRAM Uncompressed   | Basis Universal     |
++===============+=====================+=====================+=====================+=====================+=====================+
+| **128×128**   | |good| 85 KiB       | |good| 85 KiB       | |good| 21 KiB       | |good| 85 KiB       | |good| 21 KiB       |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| **256×256**   | |good| 341 KiB      | |good| 341 KiB      | |good| 85 KiB       | |good| 341 KiB      | |good| 85 KiB       |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| **512×512**   | |good| 1.33 MiB     | |good| 1.33 MiB     | |good| 341 KiB      | |good| 1.33 MiB     | |good| 341 KiB      |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| **1024×1024** | |regular| 5.33 MiB  | |regular| 5.33 MiB  | |good| 1.33 MiB     | |regular| 5.33 MiB  | |good| 1.33 MiB     |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| **2048×2048** | |bad| 21.33 MiB     | |bad| 21.33 MiB     | |regular| 5.33 MiB  | |bad| 21.33 MiB     | |regular| 5.33 MiB  |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| **4096×4096** | |bad| 85.33 MiB     | |bad| 85.33 MiB     | |bad| 21.33 MiB     | |bad| 85.33 MiB     | |bad| 21.33 MiB     |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
```

Comment:
Not sure if we should be saying "good" or "bad," might be a better idea to just give the size and let people make their own decisions. The way this is made makes it seem like you should never use 4K textures.

Reply:
The "good" or "bad" badges are mostly for visual grepping – they're not visible text on the page.